app/views/course/forum/posts/_post.html.slim

```diff
@@ -0,0 +1,26 @@
+= content_tag_for(:post, post) do
+  div.post
+    div.user
+      h5 = link_to_user(post.creator)
+
+    div.contents
+      div.pull-right
+        .timestamp
+          => format_datetime(post.created_at)
+
+      h4.title
+        a = post.title
+
+      = format_block_text(post.text)
+
+      - if defined?(show_buttons)
+        div.info
+          div.pull-right
+            - if can? :reply_topic, @topic
+              = link_to reply_course_forum_topic_post_path(@course, @forum, @topic, post),
+                        title: t('course.forum.posts.reply.tag'), class: ['btn', 'btn-primary'] do
```

Comment:
hmm... I'm not quite understand the problem. Do you mean that I should write localisation for this partial instead of using its parent view's localisation?